tests/src/Interop/MarshalAPI/DllImportUtil/DllImportUtil.cs

```diff
@@ -0,0 +1,110 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using TestLibrary;
+
+using Console = Internal.Console;
+
+namespace DllImportUtil
+{
+    public enum TestResult {
```

Comment:
I really don't see why these error codes are needed. It almost seems like an entire sub-test framework is being built here. Tests should just throw an exception with a message. All this infrastructure just makes the testing appear more complex than it needed. We also already have the `Assert` API that should be used instead of custom solutions that bifurcate having consistent tests.